cpp/tests/reductions/scan_tests.cpp

```diff
@@ -135,8 +135,18 @@ struct ScanTest : public BaseScanTest<T> {
       switch (agg->kind) {
         case aggregation::SUM: return HostType{0};
         case aggregation::PRODUCT: return HostType{1};
-        case aggregation::MIN: return std::numeric_limits<HostType>::max();
-        case aggregation::MAX: return std::numeric_limits<HostType>::lowest();
+        case aggregation::MIN:
+          if constexpr (cuda::std::numeric_limits<T>::has_infinity) {
+            return cuda::std::numeric_limits<T>::infinity();
```

Comment:
Is this supposed to be a `HostType`? Is it supposed to use `std` instead of `cuda::std`? Mixing the two seems incorrect.
```suggestion
          if constexpr (std::numeric_limits<HostType>::has_infinity) {
            return std::numeric_limits<HostType>::infinity();
```

Reply:
Oh bad. Copy+past forever issue.